src/sagemaker/tensorflow/estimator.py

```diff
@@ -537,6 +538,8 @@ def create_model(
                 If not specified and ``endpoint_type`` is 'tensorflow-serving', ``dependencies`` is
                 set to ``None``.
                 If ``endpoint_type`` is also ``None``, then the dependencies from training are used.
+            **kwargs: Additional kwargs passed to ``_create_tfs_model`` and
+                ``_create_default_model``.
```

Comment:
I think you'd be better off mentioning the model class names rather than these private methods